extensions/eventtap/init.lua

```diff
@@ -177,29 +177,40 @@ function module.rightClick(point, delay)
     module.event.newMouseEvent(module.event.types["rightMouseUp"], point):post()
 end
 
---- hs.eventtap.middleClick(point[, delay])
+--- hs.eventtap.otherClick(point[, delay][, button])
 --- Function
 --- Generates a middle mouse click event at the specified point
 ---
 --- Parameters:
----  * point - A table with keys `{x, y}` indicating the location where the mouse event should occur
----  * delay - An optional delay (in microseconds) between mouse down and up event. Defaults to 200000 (i.e. 200ms)
----
+---  * point  - A table with keys `{x, y}` indicating the location where the mouse event should occur
+---  * delay  - An optional delay (in microseconds) between mouse down and up event. Defaults to 200000 (i.e. 200ms)
+---  * button - An optional integer, default 2, between 2 and 31 specifying the button number to be pressed.  If this parameter is specified then `delay` must also be specified, though you may specify it as `nil` to use the default.
 --- Returns:
 ---  * None
 ---
 --- Notes:
----  * This is a wrapper around `hs.eventtap.event.newMouseEvent` that sends `middlemousedown` and `middlemouseup` events)
-function module.middleClick(point, delay)
+---  * This is a wrapper around `hs.eventtap.event.newMouseEvent` that sends `otherMouseDown` and `otherMouseUp` events)
+---  * For backwards compatibility, `hs.eventtap.middleClick` can also be used to invoke this function.
+---
+---  * OS X recognizes up to 32 distinct mouse buttons, though few mouse devices have more than 3.  The left mouse button corresponds to button number 0 and the right mouse button corresponds to 1;  distinct events are used for these mouse buttons, so you should use `hs.eventtap.leftClick` and `hs.eventtap.rightClick` respectively.  All other mouse buttons are coalesced into the `otherMouse` events and are distinguished by specifying the specific button with the `mouseEventButtonNumber` property, which this function does for you.
```

Comment:
Simple enough to adjust when I see if other changes are also required.

@cmsj, doing a pattern search on `docs.json` with `\s[oO][sS]\s?[xX]\s` shows that it appears 208 times in the documentation... do we care enough to open an issue to change this in the future, or do we just want to try and be cognizant of the name change going forward?